website/docs/sources/general.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Overview of sources
+slug: general
+---
+
+Sources allow you to connect authentik to an external user directory. They can also be used for social logins, using external providers such as Facebook, Twitter, etc.
```

Comment:
The commas here feel... off-balanced?  Perhaps "Sources can also be used with social login providers such as Facebook, Twitter, or GitHub."